src/utils/gfycat-sdk.umd.bundle.min.js

```diff
@@ -0,0 +1,3 @@
+/* eslint-disable */
```

Comment:
Is this available as an NPM package? Would be nice to be able to reference the original source and minify it as part of our build. I don't think this is a blocker.